include/core/parameters.h

```diff
@@ -1064,6 +1063,12 @@ namespace Parameters
     /// Maximum number of krylov vectors
     int max_krylov_vectors;
 
+    /// Enable hessians in jacobian
+    bool enable_hessians_jacobian;
```

Comment:
why is the parameter repeated twice?

Reply:
Same explanation as in the comments above.